org.eclipse.xtext.tests/src/org/eclipse/xtext/junit5/RunSuiteJUnit5Suite.java

```diff
@@ -21,6 +21,6 @@
 @UseTechnicalNames
 @SelectPackages("org.eclipse.xtext.validation.junit5")
 @SuppressWarnings("deprecation")
-public class RunSuiteJUnit5 {
+public class RunSuiteJUnit5Suite {
```

Comment:
hmmm guess the same as https://github.com/eclipse/xtext-core/compare/cd_junit5_call_4?expand=1